Makefile

```diff
@@ -2904,6 +2931,43 @@ distclean: clean toolsclean
 dist:
 	git archive --format zip --output $(CLIENTBIN)-$(VERSION).zip HEAD
 
+#############################################################################
+# INSTALL (only for Linux platforms)
+#############################################################################
+
+# Shell scripts for running binaries
+
+$(B)/$(CLIENTBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(CLIENTBIN) "$$@"' >> $@
+
+$(B)/$(SERVERBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(SERVERBIN) "$$@"' >> $@
+
+# Install the .desktop, icon files, license, etc.
+install: release
+ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu"))
+	$(echo_cmd) "Installing for Linux platform in $(COPYBINDIR) and $(PREFIX)"
+	@$(INSTALL) -d $(PREFIX)/bin "$(PREFIX)/share/metainfo/" \
+			"$(PREFIX)/share/licenses/$(PACKAGE)/" "$(PREFIX)/share/applications/" \
+			"$(PREFIX)/share/icons/hicolor/128x128/apps" "$(COPYBINDIR)"
+	@cd $(BR) && for file in $(NAKED_TARGETS); do \
+		$(INSTALL) -D $$file $(COPYBINDIR)/$$file; \
+	done
+	$(INSTALL) -D -m755 $(BR)/$(CLIENTBINSH) $(PREFIX)/bin/tremulous-grangerhub
+	$(INSTALL) -D -m755 $(BR)/$(SERVERBINSH) $(PREFIX)/bin/tremded-grangerhub
+	$(INSTALL) -D -m644 "misc/Tremulous-Grangerhub.png" "$(PREFIX)/share/icons/hicolor/128x128/apps/"
+	$(INSTALL) -D -m644 "misc/Tremulous-Grangerhub.desktop" \
```

Comment:
Wait... Why are we installing the file with different (upper case) name? It does not make sense.